great_expectations_cloud/agent/config.py

```diff
@@ -7,8 +7,9 @@
 class GxAgentEnvVars(BaseSettings):
     # pydantic will coerce this string to AnyUrl type
     gx_cloud_base_url: AnyUrl = CLOUD_DEFAULT_BASE_URL  # type: ignore[assignment]
-    gx_cloud_organization_id: str
-    gx_cloud_access_token: str
+    gx_cloud_organization_id: str = ""
+    gx_cloud_access_token: str = ""
```

Comment:
I could use a second set of eyes on this. Without adding the default values, `test_logging_config` was failing with this error:

 ```
______________________________________________ ERROR collecting tests/test_logging_cfg.py _______________________________________________
tests/test_logging_cfg.py:13: in <module>
    from great_expectations_cloud.logging.logging_cfg import (
great_expectations_cloud/logging/logging_cfg.py:21: in <module>
    SERVICE_NAME: Final[str] = GxAgentEnvVars().service_name
great_expectations_cloud/agent/config.py:21: in __init__
    super().__init__(**overrides)
.venv/lib/python3.10/site-packages/pydantic/v1/env_settings.py:40: in __init__
    super().__init__(
.venv/lib/python3.10/site-packages/pydantic/v1/main.py:341: in __init__
    raise validation_error
E   pydantic.v1.error_wrappers.ValidationError: 2 validation errors for GxAgentEnvVars
E   gx_cloud_organization_id
E     field required (type=value_error.missing)
E   gx_cloud_access_token
E     field required (type=value_error.missing)
```

I wasn't sure how else to fix this

Reply:
yea we see this in the Runner code as well. There we use `monkeypatch` to set values to the env vars.  @anthonyburdi  I know you worked on ENV vars and tests recently.. is this a viable approach here too?

https://github.com/greatexpectationslabs/gx-runner/blob/dd2334ef3a5f1bc885c9827a75e867e53adfd434/tests/test_runner.py#L55

Reply:
Good call @Shinnnyshinshin - yes we used `monkeypatch` to set these env variables. We don't want to rely on the state of the environment for unit tests. Take a look at this PR for some examples: https://github.com/great-expectations/cloud/pull/318/files. You may want to create a fixture like the `set_required_env_vars` fixture to use in all the tests in `tests/test_logging_cfg.py`

Reply:
Yes I would use `monkeypatch.setenv` for this.
And ideally create a fixture that sets all the needed env vars.

https://docs.pytest.org/en/4.6.x/monkeypatch.html#monkeypatching-environment-variables

Reply:
Thanks, all. I have been trying different variations of `monkeypatch` but am having issues b/c the tests are erroring on the `logging_cfg` import, and I can't figure out how to mock this sort of import.

Reply:
Thanks!

> Where does this logging_cfg.SERVICE_NAME variable actually get used?

I was super confused about this; thought it was some magic variable name or something that was causing the service name to show in the logs.

Looking at this more closely, though, it looks like we *not* using this variable at all. Instead, we are passing in a custom log tag for `service_name` in the Kubernetes manifest. 

So TL;DR, it looks ike none of these changes are actually needed. When we run the GX Runner agent, we can just update the manifest to use `gx-runner` as the `service_name` and then the logs pushed to Datadog will include the appropriate service name.

I'll close out this PR. Thanks again for the feedback, much appreciated.
cc @allensallinger

Reply:
FWIW, I do think it would be valuable for a user to be able to control the service name (via env var or command line argument).

An organization likely may need to deploy multiple agents to connect to all of its different data sources (or would want to for security reasons).
In that case, being able to customize the `service-name` can be helpful to distinguish them.

Reply:
The agent cli accepts an argument for `--custom-log-tags` that @rreinoldsc added in #191 , so we should be good there!


https://github.com/great-expectations/cloud/blob/f5ffaab324ca770c8be2303843b7baecb2e5b950/great_expectations_cloud/agent/cli.py#L56